### ISSUE TITLE
Project: re-train ECAPA-TDNN on adults' and children's voices

### ISSUE THREAD

**Issue by fabiocat93** [sensein/senselab#45]
### Description

Train ECAPA-TDNN on adults' and children's voices

### Tasks

- [ ] explore the impact of training data window on speaker diarization performance (in a child-adult setting) and on speaker verification 
- [ ] explore the impact of including children's data on speaker diarization performance (in a child-adult setting) and on speaker verification

**Comment by fabiocat93** [comment]
https://docs.google.com/document/d/1wGjFpzVY6hswNJXwLfep5LmpJi2sWnPUI4-_XOlTewI/edit

**Comment by fabiocat93** [comment]
@900miles, do you happen to have any updates with this?

**Comment by 900miles** [comment]
Yes; ecapa was trained with a shorter context window (subpart 1) and has been evaluated on speaker verification. We are working on evaluating it for diarization now. That is the context for the CHILDES dataset project, as well as the generating synthetic dataset project I am working on currently. Once we have some good datasets, I can use those to evaluate the model for diarization. We can also use those datasets for subpart 2 afterwards.

**Comment by fabiocat93** [comment]
@900miles are you still working on this? I will close this issue since this looks more like a separate project than a senselab pipeline/feature. Feel free to re-open it if you think otherwise

**Comment by 900miles** [comment]
Yes it's a separate project and possibly redundant given the quality of nvidia's sortformer model.
